Source/WebCore/platform/RectEdges.h

```diff
@@ -151,6 +151,8 @@ template<typename T> class RectEdges {
 
     bool operator==(const RectEdges<T>&) const = default;
 
+    void unite(const RectEdges<T>&);
```

Comment:
This is a little odd to have here because type T might not be something you can call `max()` on.

Reply:
Yes but:

1. For `RectEdges`, we define the `operator+`. This operator would not compile for `RectEdges<LayoutUnit>` if `operator+` was not defined `LayoutUnit`.
2. This is a template function. So the source will compile for `RectEdges<X>` even if type `X` does not define the comparison operators. The compiler will complain only if you call `maximum(x1, x2)`